### PR TITLE
Release v5.6.0 rc1

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

File: doc/release-notes/release-notes-5.6.0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+Notable changes
+===============
```
Comment (nuttycom) [inline doc/release-notes/release-notes-5.6.0-rc1.md:2]: Need to add the RPC changes to the release notes.

**Comment by nuttycom** [comment]
[diff from v5.5.1](https://github.com/zcash/zcash/compare/v5.5.1..85816086adc03e1c995c6da757d02b49dbb3f7a3)

**Review by daira — APPROVED** [review]
utACK, modulo adding release notes for RPC changes.

**Review by ebfull — APPROVED** [review]

**Comment by nuttycom** [comment]
I will make the updates to the release notes prior to the release.

File: depends/packages/native_rust.mk [hunk 2]
```diff
@@ -3,16 +3,16 @@ package=native_rust
 # ./contrib/devtools/update-rust-hashes.sh
 # The Rust compiler should use the same LLVM version as the Clang compiler; you
 # can check this with `rustc --version -v`.
-$(package)_version=1.69.0
+$(package)_version=1.70.0
```
Comment (str4d) [inline depends/packages/native_rust.mk:6]: NACK. Rust 1.70 migrates to LLVM 16; this PR updated Rust without updating Clang.

**Pull request merged**